packages/pipes.sh/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pipeseroni/pipes.sh
+TERMUX_PKG_DESCRIPTION="Animated pipes terminal screensaver"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@Efreak"
+TERMUX_PKG_VERSION=1.3.0+ #if there's ever a new release, switch to it
+TERMUX_PKG_SRCURL=https://github.com/pipeseroni/pipes.sh/archive/refs/heads/master.zip # in the meantime, master has a bunch of changes
```

Comment:
Source code downloads must be deterministic. Please use tags (if possible) or specific commit. `master.zip` will change over time and will make git history for that package useless.